src/huntsman/pocs/utils/focus.py

```diff
@@ -13,16 +13,20 @@ class AutofocusSequence(PanBase):
     the 'is_finished' property.
     """
 
-    def __init__(self, position_min, position_max, position_step, bit_depth, mask_threshold=0.3,
-                 extra_focus_steps=5, mask_dilations=10, merit_function_name="vollath_F4",
-                 merit_function_kwargs=None, image_dtype=np.float32, **kwargs):
+    def __init__(self, position_min, position_max, position_step, bit_depth,
+                 dark_mask_threshold=0.3, extra_focus_steps=5, mask_dilations=10,
+                 merit_function_name="vollath_F4", merit_function_kwargs=None,
+                 image_dtype=np.float32, mask_threshold=0.9, **kwargs):
         """
         Args:
             position_min (int): The minimal focus position.
             position_max (int): The maximal focus position.
             position_step (int): The step in focus position.
             bit_depth (astropy.units.Quantity): The bit depth of the images.
-            mask_threshold (float, optional): The staturation masking threshold.
+            mask_threshold (float): The staturation masking threshold to apply to focus images.
+                Default: 0.9.
+            dark_mask_threshold (float, optional): The staturation masking threshold to eliminate
+                hot pixels from initial dark frame.
```

Comment:
The purpose of these might be clearer if they are called something like "saturation_mask_threshold" and "hot_pixel_mask_threshold".